shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -492,88 +472,46 @@ - (FlutterViewController*)viewController {
   return _viewController.get();
 }
 
-- (FlutterPlatformPlugin*)platformPlugin {
-  return _platformPlugin.get();
-}
 - (std::shared_ptr<flutter::PlatformViewsController>&)platformViewsController {
   return _platformViewsController;
 }
-- (FlutterTextInputPlugin*)textInputPlugin {
-  return _textInputPlugin.get();
-}
-- (FlutterUndoManagerPlugin*)undoManagerPlugin {
-  return _undoManagerPlugin.get();
-}
-- (FlutterRestorationPlugin*)restorationPlugin {
-  return _restorationPlugin.get();
-}
-- (FlutterMethodChannel*)localizationChannel {
-  return _localizationChannel.get();
-}
-- (FlutterMethodChannel*)navigationChannel {
-  return _navigationChannel.get();
-}
-- (FlutterMethodChannel*)restorationChannel {
-  return _restorationChannel.get();
-}
-- (FlutterMethodChannel*)platformChannel {
-  return _platformChannel.get();
-}
-- (FlutterMethodChannel*)textInputChannel {
-  return _textInputChannel.get();
-}
-- (FlutterMethodChannel*)undoManagerChannel {
-  return _undoManagerChannel.get();
-}
-- (FlutterMethodChannel*)scribbleChannel {
-  return _scribbleChannel.get();
-}
-- (FlutterMethodChannel*)spellCheckChannel {
-  return _spellCheckChannel.get();
-}
-- (FlutterBasicMessageChannel*)lifecycleChannel {
-  return _lifecycleChannel.get();
-}
-- (FlutterBasicMessageChannel*)systemChannel {
-  return _systemChannel.get();
-}
-- (FlutterBasicMessageChannel*)settingsChannel {
-  return _settingsChannel.get();
-}
-- (FlutterBasicMessageChannel*)keyEventChannel {
-  return _keyEventChannel.get();
-}
 
 - (NSURL*)observatoryUrl {
-  return [_publisher.get() url];
+  return [self.publisher url];
```

Comment:
Optional here and below: `url` is a property on `FlutterDartVMServicePublisher`, so this can be `self.publisher.url`.

Reply:
Good call - I think my brain had turned to mush at this point. Done.